technical-specs/001-Identity-MVP-Spec/001-Identity-Spec.md

```diff
@@ -0,0 +1,125 @@
+
+# App Center Identity Spec 
+
+This spec details the App Center Identity service that we will launch a public preview MVP.
+
+## Overview 
+
+### TL;DR 
+The goal is to deliver an identity management service in App Center that allows app developers to: 
+
+1. Create and manage an Azure AD B2C resource/tenant in App Center 
+2. Configure the Azure AD B2C resource/tenant for the following identity approaches in order of priority: Google, Facebook, Email/Password, Microsoft 
+3. View and search for user identities, and the ability to edit in Azure 
+4. Use the provided identity to integrate with other services in App Center: 
+   - Crashes and analytics events (i.e. Users of the app will be tagged to their crash/analytics data) 
+   - Push notifications (i.e. App developers can send push notifications to specific users)
+   - Data Storage/GDPR 
+
+## User scenarios 
+
+### Overview 
+
+We are prioritizing the following platforms and providers: 
+- Platforms: Android, iOS, Xamarin, React Native
+- Provider configuration in App Center Portal (in order of priority):  
+  - Google,  
+  - Facebook, 
+  - Email/password,  
+  - Microsoft
+
+- Additional providers can be configured through the B2C tenant directly
+
+Additional platforms will be targeted in the future:
+- Windows and PWA/SPA Support
```

Comment:
We have found a decent amount of app developers will build macOS and iOS from the same code base. It may be worth considering macOS. We have some developers who cannot use the latest offering for iOS exactly for this reason, it MUST also support macOS.

Reply:
Ah I see! Got it. We'll keep that in mind. I'd imagine we will get user feedback on this in the future.